src/main/java/frc/robot/commands/DriveUsingController.java

```diff
@@ -7,27 +7,38 @@
  
 package frc.robot.commands;
 
+import com.nrg948.preferences.RobotPreferences;
+import com.nrg948.preferences.RobotPreferences.DoubleValue;
 import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.wpilibj.GenericHID.RumbleType;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.subsystems.Elevator;
 import frc.robot.subsystems.Subsystems;
 import frc.robot.subsystems.Swerve;
+import frc.robot.util.RateLimiter;
 
 /** A command that enables the driver to drive the robot using an Xbox controller. */
 public class DriveUsingController extends Command {
   private static final double RUMBLE_MIN_G = 1.0;
   private static final double RUMBLE_MAX_G = 8.0;
-
+  public static final RobotPreferences.DoubleValue minPower =
```

Comment:
We generally treat preferences as constants and should SCREAMING_SNAKE_CASE for their names.